src/Microsoft.AspNetCore.Razor/CodeGenerators/GeneratedTagHelperContext.cs

```diff
@@ -29,6 +29,8 @@ public GeneratedTagHelperContext()
             MarkAsHtmlEncodedMethodName = "Html.Raw";
             StartTagHelperWritingScopeMethodName = "StartTagHelperWritingScope";
             EndTagHelperWritingScopeMethodName = "EndTagHelperWritingScope";
+            BeginWriteTagHelperAttributeMethodName = "BeginWriteTagHelperAttribute";
```

Comment:
Add these setters to the MvcRazorHost. [Example](https://github.com/aspnet/Mvc/blob/dev/src/Microsoft.AspNetCore.Mvc.Razor.Host/MvcRazorHost.cs#L107-L108)